network/horizon/api-reference/errors/error-handling.mdx

```diff
@@ -0,0 +1,184 @@
+
+#### Synchronous Transaction Submission
+
+Due to the blocking nature of this endpoint, things are a little different compared to the asynchronous strategy. There are 3 possible scenarios that clients can encounter:
+
+1. The submission is successful and Horizon returns the transaction response back. This is the happy path and clients do not need to do anything but wait for Horizon's response.
+2. Stellar-Core sends back an `ERROR` response from the submission. Clients should consult the attached error message and retry the submission again.
+3. Timeouts: Horizon may respond with a `504` HTTP code. This response is not an error but a warning that your transaction hasn't been accepted by the network yet. There could be many possible reasons for the timeout, the most common of which is network congestion, but it could also be due to other transient issues.
+   1. **Polling the transaction hash**: Use the transaction hash in the timeout response and poll the [GET /transactions/:transaction_hash](/network/horizon/api-reference/resources/retrieve-a-transaction) endpoint to see if it successfully makes it into a ledger.
+   2. **Timebounds**: Timebounds are optional but **highly** recommended as they put a definitive time limit on the transaction's finality - after it times out, you will know for sure whether it made it into a ledger.
+   3. **Resubmitting the transaction**: If the transaction status poll has not returned anything or if your transaction has timed out based on the time bounds you specified, you should attempt a resubmission of the same transaction.
+
+Horizon will return a valid transaction response for any resubmissions for successful transactions. However, do note that resubmitting a transaction is only safe when it is unchanged - same operations, signatures, sequence number, etc.... Be careful when working around an error that does require changes to the transaction. It can cause duplicate transactions, which can cause problems - double payments, incorrect trustlines, and more. If you continue to face timeouts on retries, consider using a fee-bump transaction to get into the ledger (after the initial transaction's timebound expires) or increasing the maximum fee you’re willing to pay. Read up on [Surge Pricing and Fee Strategies](docs/learn/fundamentals/fees-resource-limits-metering) for more details.
+
+### Example: Using Time Bounds
+You submit a transaction, and it enters the queue of the Stellar network, but Horizon crashes while giving you a response. Uncertain about the transaction status, you resubmit the transaction (with no changes!) until either a. Horizon comes back up to give you a reply or b. your time bounds are exceeded.
+
+There are only two possible results to this scenario: either the transaction makes it into the ledger (exactly once) and Horizon gives you the response, or the transaction never makes it out of the queue, and you receive the corresponding `tx_too_late` response.
+
+Example implementation:
```

Comment:
We have two examples but they both use the sync endpoint, can we add an example for the async endpoint?